src/content/developers/docs/networks/index.md

```diff
@@ -65,7 +67,7 @@ A proof-of-authority testnet that works across clients; an ideal testnet for app
 
 _Note, [the Ropsten testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Görli._
 
-A proof-of-work testnet. It only serves historical relevance and is mainly used by core developers for testing protocol upgrades.
+Ropsten was a proof-of-work testnet that went through The Merge to proof-of-stake in May 2022. It can be used to test applications on a merged network, but it is not expected to be maintained long term and is likely to deprecated before summer 2023.
```

Comment:
```suggestion
Ropsten was a proof-of-work testnet that went through [The Merge to proof-of-stake in June 2022](https://blog.ethereum.org/2022/05/30/ropsten-merge-announcement/). It can be used to test applications on a merged network, but it is not expected to be maintained long term and is likely to deprecated before summer 2023.
```